### PR TITLE
Handlers/Filters for handshake

### PR DESCRIPTION
**Pull request by maxpowa**
Not really sure if Filter or Handler is better. Also not sure where the common interface should go.

### CODE REVIEW

**Comment by asiekierka** [comment]
LGTM

File: mappings/net/minecraft/server/DummyFilter.mapping [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CLASS none/me net/minecraft/server/DummyFilter
```
Comment (asiekierka) [inline mappings/net/minecraft/server/DummyFilter.mapping]: Just Filter? Not ConnectionFilter?

Comment (maxpowa) [reply mappings/net/minecraft/server/DummyFilter.mapping]: I can change it, should I change the interface to IConnectionFilter as well then?

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/sortme/IConnectionFilter.mapping [hunk 2]
```diff
@@ -0,0 +1,3 @@
+CLASS none/jp net/minecraft/sortme/IConnectionFilter
```
Comment (shadowfacts) [inline mappings/net/minecraft/sortme/IConnectionFilter.mapping]: Maybe put this in the `network` package?

Comment (maxpowa) [reply mappings/net/minecraft/sortme/IConnectionFilter.mapping]: Done

**Review by shadowfacts — APPROVED** [review]

**Pull request merged**